tc/core/cpu/cpu_mapping_options.h

```diff
@@ -17,7 +17,7 @@
 
 #include <mapping_options.pb.h>
```

Comment:
this one should be, IMO, tc/proto/mapping_options.pb , but we can do it in a separate PR, of course.